src/View/mainpage/DrawROIWindow/UIDrawROIWindow.py

```diff
@@ -802,16 +828,16 @@ def onDrawClicked(self):
                 self.is_drawing.disconnect(self.drawingROI.set_is_drawing)
         logging.debug("onDrawClicked finished")
 
-    def onFillClicked(self):
+    def onFillClicked(self, is_3d):
         """
-        Function triggered when the Fill button is pressed from the menu.
+        Function triggered when the Fill or 3d Fill button is pressed from the menu.
         """
         logging.debug("onFillClicked started")
```

Comment:
When a user clicks a 3d fill, this is gonna get spammed. Might be good to include some useful info like the current slice?